sections/collage.liquid

```diff
@@ -53,10 +53,10 @@
               <div class="collage-content card{% if block.settings.secondary_background %} card-colored color-background-1{% endif %}{% if block.settings.image_padding %} collage-card-spacing{% endif %}" {{ block.shopify_attributes }}>
                 {%- if featured_media != blank -%}
                   <div class="collage-content__info card-information card-information__wrapper">
-                    <h3 class="card-information__text h4"><a href="{{ block.settings.product.url }}" class="full-unstyled-link">{{ block.settings.product.title | escape }}</a></h3>
+                    <div class="card-information__text h4"><a href="{{ block.settings.product.url }}" class="full-unstyled-link">{{ block.settings.product.title | escape }}</a></div>
```

Comment:
@ludoboludo What's the rationale behind this change?

Reply:
Oh! The reason why I was doing this, is because of some feedback we received as an issue for which I was curious to hear your take on actually @svinkle: https://github.com/Shopify/dawn/issues/682

Because heading elements are described as section headings ([MDZ docs](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/Heading_Elements)), I was wondering if it made sense to use them for a product/collection card. When navigating via screen reader and using the heading list available on the page, does it make sense to be provided with the title of those cards?

I don't have a strong opinion myself, I'm just wondering what makes the most sense semantically.

Reply:
Collection cards only.